### PR TITLE
Fix fastq sync prefixes

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

File: biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala [hunk 1]
```diff
@@ -183,4 +193,21 @@ object FastqSync extends ToolCommand {
       BWriter.close()
     }
   }
+
+  def findR1R2Subfixes(fastqR1: File, fastqR2: File): (String, String) = {
```
Comment (sndrtj) [inline biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala]: Could you please add a docstring as to what this does.

File: biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala [hunk 2]
```diff
@@ -164,7 +172,9 @@ object FastqSync extends ToolCommand {
 
     val commandArgs: Args = parseArgs(args)
 
-    val refReader = new FastqReader(commandArgs.refFastq)
+    val subFixes = findR1R2Subfixes(commandArgs.refFastq1, commandArgs.refFastq2)
```
Comment (sndrtj) [inline biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala]: This value does not seem to be used anywhere.

**Review by sndrtj — CHANGES_REQUESTED** [review]

File: biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala (hunk 2 above)
Comment (ffinfo) [reply biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala]: Good point, something went wrong there

File: biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala (hunk 1 above)
Comment (ffinfo) [reply biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/FastqSync.scala]: Added this

**Pull request merged**